docs/configuration/flag_configuration.md

```diff
@@ -100,46 +159,149 @@ Example of an invalid configuration:
 A targeting rule **must** be valid JSON.
 Flagd uses a modified version of [JSON Logic](https://jsonlogic.com/), as well as some custom pre-processing, to evaluate these rules.
 The output of the targeting rule **must** match the name of one of the variants defined above.
-If an invalid or null value is is returned by the targeting rule, the `defaultVariant` value is used.
-If no targeting rules are defined, the response reason will always be `STATIC`, this allows for the client side caching of these flag values, this behavior is described [here](../other_resources/caching.md).
+If an invalid or null value is returned by the targeting rule, the `defaultVariant` value is used.
+If no targeting rules are defined, the response reason will always be `STATIC`, this allows for the flag values to be cached, this behavior is described [here](../other_resources/caching.md).
 
-The [JSON Logic playground](https://jsonlogic.com/play.html) is a great way to experiment with new targeting rules.
-The following example shows how a rule could be configured to return `binet` when the email (which comes from evaluation context) contains `@faas.com`.
-If the email wasn't included in the evaluation context or doesn't contain `@faas.com`, null is returned and the `defaultVariant` is used instead.
+#### Evaluation Context
 
-<details>
-  <summary>Click here to see how this targeting rule would look in the JSON Logic playground.</summary>
+Evaluation context is included as part of the evaluation request.
+For example, when accessing flagd via HTTP, the POST body may look like this:
 
-1. Open the [JSON Logic playground](https://jsonlogic.com/play.html) in your favorite browser
-1. Add the follow JSON as the `Rule`:
+```json
+{
+  "flagKey": "booleanFlagKey",
+  "context": {
+    "email": "noreply@example.com"
+  }
+}
+```
 
-    ```json
-    {
-      "if": [
-        {
-          "in": [
-            "@faas.com",
-            {
-              "var": ["email"]
-            }
-          ]
-        },
-        "binet",
-        null
-      ]
-    }
-    ```
+The evaluation context can be accessed in targeting rules using the `var` property followed the evaluation context property name.
```

Comment:
```suggestion
The evaluation context can be accessed in targeting rules using the `var` operation followed the evaluation context property name.
```

You could go with `keyword` or just say "by using `var`" - but I think "property" is wrong here.